RNAdam-core/src/main/scala/org/bdgenomics/RNAdam/algorithms/quantification/Index.scala

```diff
@@ -68,7 +68,7 @@ object Index extends Serializable with Logging {
   private[quantification] def extractKmers(transcript: Transcript,
                                            kmerLength: Int, 
                                            referenceFile: ReferenceFile): List[String] = {
-    referenceFile.extract(transcript.region).inits.flatMap(_.tails).filter(_.length == kmerLength).toList
+    referenceFile.extract(transcript.region).sliding(kmerLength).toList
```

Comment:
I think you can eliminate the toList call here. You might need to do `toIterable` and return an `Iterable[String]`, but that'd be preferable over a list.